src/styles/blocks/container.scss

```diff
@@ -0,0 +1,17 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  justify-content: center;
+
+  &__box {
+    height: 300px;
+    text-align: center;
+    line-height: 300px;
+  }
+}
+
+@media(min-width:1200px) {
+  .container {
+    grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
+  }
+}
```

Comment:
Bro, this property makes your snake stretch too long on larger screens. It should occupy 3 columns at most. Tame it.

For that, you need to do something to that `auto-fill` property.